src/tplr/config.py

```diff
@@ -28,73 +28,51 @@
-                        "R2_GRADIENTS_READ_SECRET_ACCESS_KEY"
-                    ),
-                },
-                "write": {
-                    "access_key_id": os.environ.get("R2_GRADIENTS_WRITE_ACCESS_KEY_ID"),
-                    "secret_access_key": os.environ.get(
-                        "R2_GRADIENTS_WRITE_SECRET_ACCESS_KEY"
-                    ),
-                },
-            },
-        },
-        "aggregator": {
-            "account_id": os.environ.get("R2_AGGREGATOR_ACCOUNT_ID"),
-            "name": os.environ.get("R2_AGGREGATOR_BUCKET_NAME"),
-            "credentials": {
-                "read": {
-                    "access_key_id": os.environ.get("R2_AGGREGATOR_READ_ACCESS_KEY_ID"),
-                    "secret_access_key": os.environ.get(
-                        "R2_AGGREGATOR_READ_SECRET_ACCESS_KEY"
-                    ),
-                },
-                "write": {
-                    "access_key_id": os.environ.get(
-                        "R2_AGGREGATOR_WRITE_ACCESS_KEY_ID"
-                    ),
-                    "secret_access_key": os.environ.get(
-                        "R2_AGGREGATOR_WRITE_SECRET_ACCESS_KEY"
-                    ),
-                },
+def format_bucket_secrets(base_secret_name: str) -> dict[str, str]:
+    """
+    Given the typical patterns for our os vars, retrieve per key
+
+    Args:
+        base_secret_name: the root of all other keys
+
+    Returns:
+        the formatted dict for comms ingestion
+    """
+
+    return {
+        "account_id": os.getenv(f"{base_secret_name}_ACCOUNT_ID"),
+        "name": os.getenv(f"{base_secret_name}_BUCKET_NAME"),
+        "credentials": {
+            "read": {
+                "access_key_id": os.getenv(f"{base_secret_name}_READ_ACCESS_KEY_ID"),
+                "secret_access_key": os.getenv(
+                    f"{base_secret_name}_READ_SECRET_ACCESS_KEY"
+                ),
             },
-        },
-        "dataset": {
-            "account_id": os.environ.get("R2_DATASET_ACCOUNT_ID"),
-            "name": os.environ.get("R2_DATASET_BUCKET_NAME"),
-            "credentials": {
-                "read": {
-                    "access_key_id": os.environ.get("R2_DATASET_READ_ACCESS_KEY_ID"),
-                    "secret_access_key": os.environ.get(
-                        "R2_DATASET_READ_SECRET_ACCESS_KEY"
-                    ),
-                },
-                "write": {
-                    "access_key_id": os.environ.get("R2_DATASET_WRITE_ACCESS_KEY_ID"),
-                    "secret_access_key": os.environ.get(
-                        "R2_DATASET_WRITE_SECRET_ACCESS_KEY"
-                    ),
-                },
+            "write": {
+                "access_key_id": os.getenv(f"{base_secret_name}_WRITE_ACCESS_KEY_ID"),
+                "secret_access_key": os.getenv(
+                    f"{base_secret_name}_WRITE_SECRET_ACCESS_KEY"
+                ),
```

Comment:
We don't always require write keys. But I guess this is fine because we are using `get` and default to `None`. It is the same behavior as before. Just pointing that out.